src/service/student_service.ts

```diff
@@ -23,24 +24,49 @@ export class StudentService {
             throw new Error(`Error! status: ${response.status}`);
         }
         const result = (await response.json());
-        const getResult = <Student[]>JSON.parse(JSON.stringify(result, null, 4));
-
-        this._students = getResult as Array<Student>;
+        this._students = <Student[]>JSON.parse(JSON.stringify(result));
+        return this._students;
+    }
 
-        console.log(result);
+    getStudents(): Array<Student> {
         return this._students;
     }
 
+    async increaseAbsent(studentId: number) {
+        const response = await fetch(apiUrl + '/students/' + studentId + '/absent', {   
+            method: 'PUT',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                id: studentId,
+                absen: 1
+             })
+        });
+
+        if (!response.ok) {
+            throw new Error(response.statusText);
+        }
+
+        const result = await response.json();
+        console.log('result is: ', JSON.stringify(result));
+        const student = this.getStudent(studentId);
+        if (student) {
+            student.absent = result.absent;
+        }
+    }
 
     async addStudent(name: string, surname: string) {
-        const response = await fetch(apiUrl + '/students', {
+        const response = await fetch(apiUrl + '/students/' + this._sequence + '/' + name + '/' + surname, {
```

Comment:
POST operasyonu için api url'i nasıl olmalı öğrenilip buna göre tekrar düzeler misin?